src/platform.rs

```diff
@@ -414,20 +582,6 @@ mod tests {
         );
     }
 
-    #[test]
-    fn test_setup_partition() {
```

Comment:
Was this moved intentionally?

Reply:
Not intentionally, will move it back to clean up the diff